modules/core/src/main/java/org/apache/ignite/IgniteSet.java

```diff
@@ -79,6 +79,7 @@
 
     /**
      * Removes this set.
+     * To check state you could use {@link #removed()}.
```

Comment:
What about "In order to check the state it's possible to use  {@link #removed()}." ? Up to you.